lib/sasl/src/systools_make.erl

```diff
@@ -1904,8 +1904,10 @@ del_tar(Tar, TarName) ->
     file:delete(TarName).
 
 add_to_tar(Tar, FromFile, ToFile) ->
-    case erl_tar:add(Tar, FromFile, ToFile, [compressed, dereference]) of
+    case catch erl_tar:add(Tar, FromFile, ToFile, [compressed, dereference]) of
```

Comment:
There was a good reason, but I'm having trouble recalling specifically what it was - I think it may have been dialyzer related, I'll play around with removing it and see if I can reproduce the reason I added it. If it does need to be kept, I'll update it to use `try`